libpod/container.go

```diff
@@ -58,6 +63,8 @@ type Container struct {
 	valid   bool
 	lock    sync.Mutex
 	runtime *Runtime
+
+	mounts []string
```

Comment:
Do these need to be in the DB-backed state? We potentially have another process removing the container (e.g. 'kpod rm -f' being run on a running attached container), which will need these

Reply:
It probably should be, that would eliminate my hack from below.